libs/components/modals/src/lib/modules/modal/modal-footer.component.ts

```diff
@@ -9,4 +12,14 @@ import { Component, ViewEncapsulation } from '@angular/core';
   styleUrls: ['./modal-footer.component.scss'],
   encapsulation: ViewEncapsulation.None,
 })
-export class SkyModalFooterComponent {}
+export class SkyModalFooterComponent {
+  public formErrors: SkyModalError[] | undefined;
+
+  #adapterSvc = inject(SkyModalComponentAdapterService);
```

Comment:
The purpose of adapter classes is to isolate DOM manipulation from the rest of the code. Adding a non-DOM-manipulating method or property to it means if we ever want to swap out all the DOM manipulation logic with another service, we'd have to duplicate the non-DOM-manipulation logic. I suggest creating a new `SkyModalErrorsService` with the new property and providing it in `SkyModalComponent`.

Reply:
added